### PR TITLE
Update to latest upstream.

### PR DESCRIPTION
**Pull request by hacktivist123**
## Description
Please describe your changes in detail according to the information below

## Related Issue
This project only accepts pull requests related to open issues.
- If suggesting a new feature or change, please discuss it in an issue first
- If fixing a bug, there should be an issue describing it with steps to reproduce it following the bug report guide
- If you're suggesting a feature, please follow the feature request guide by clicking on issues

## Motivation and Context
Why is this change required? What problem does it solve?
Improve the template all round.

## How Has This Been Tested?
Please describe in detail how you tested your changes. Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code, etc.

## Types of changes
What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
 Go over all the following list, and put an `x` in all the boxes that apply. If you're unsure about what any of these mean, don't hesitate to ask. We're here to help!

- [x] I have read the contribution guide
- [x] I have created an issue following the issue guide
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,447 @@
...
+</p>
+
+<p align="center">
+<a href="https://github.com/platformsh-templates/flask/issues">
+<img src="https://img.shields.io/github/issues/platformsh-templates/flask.svg?style=for-the-badge&labelColor=f4f2f3&color=ffd9d9&label=Issues" alt="Open issues" />
+</a>&nbsp&nbsp
+<a href="https://github.com/platformsh-templates/flask/pulls">
+<img src="https://img.shields.io/github/issues-pr/platformsh-templates/flask.svg?style=for-the-badge&labelColor=f4f2f3&color=ffd9d9&label=Pull%20requests" alt="Open PRs" />
+</a>&nbsp&nbsp
+<a href="https://github.com/platformsh-templates/drupal9/blob/master/LICENSE">
```
Comment (chadwcarlson) [inline README.md:34]:
```suggestion
<a href="https://github.com/pallets/flask/blob/main/LICENSE.rst">
```

File: README.md [hunk 2]
```diff
@@ -1,32 +1,447 @@
...
+<img src="https://img.shields.io/static/v1?label=License&message=MIT&style=for-the-badge&labelColor=f4f2f3&color=ffd9d9" alt="License" />
```
Comment (chadwcarlson) [inline README.md:35]:
```suggestion
<img src="https://img.shields.io/static/v1?label=License&message=BSD3&style=for-the-badge&labelColor=f4f2f3&color=ffd9d9" alt="License" />
```

File: README.md [hunk 3]
```diff
@@ -1,32 +1,447 @@
...
+
+### Deploy
+
+#### Quickstart
+
+The quickest way to deploy this template on Platform.sh is by clicking the button below.
+This will automatically create a new project and initialize the repository for you.
+
+<p align="center">
+    <a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/drupal9/.platform.template.yaml&utm_content=drupal9&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
```
Comment (chadwcarlson) [inline README.md:82]:
```suggestion
    <a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/flask/.platform.template.yaml&utm_content=flask&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
```

File: README.md [hunk 4]
```diff
@@ -1,32 +1,447 @@
...
+</details>
+
+<details>
+<summary>Integrate with a GitHub repo and deploy pull requests</summary>
+<!-- <blockquote>
+<br/> -->
+
+1. Create a free trial:
+
+    [Register for a 30 day free trial with Platform.sh](https://auth.api.platform.sh/register). When you have completed signup, select the **Create from scratch** project option. Give you project a name, and select a region where you would like it to be deployed. As for the _Production environment_ option, make sure to match it to whatever you have set at `https://YOUR_NAMESPACE/nextjs-drupal`.
```
Comment (chadwcarlson) [inline README.md:157]:
```suggestion
    [Register for a 30 day free trial with Platform.sh](https://auth.api.platform.sh/register). When you have completed signup, select the **Create from scratch** project option. Give you project a name, and select a region where you would like it to be deployed. As for the _Production environment_ option, make sure to match it to whatever you have set at `https://YOUR_NAMESPACE/flask`.
```

**Comment by hacktivist123** [comment]
@chadwcarlson Python 3.9 is known to have a security issue that’s why I used 3.8 

https://www.cvedetails.com/vulnerability-list/vendor_id-10210/product_id-18230/version_id-651294/Python-Python-3.9.0.html

**Review by hacktivist123 — COMMENTED** [review]
https://www.cvedetails.com/vulnerability-list/vendor_id-10210/product_id-18230/version_id-651294/Python-Python-3.9.0.html

**Comment by chadwcarlson** [comment]
@hacktivist123 if anything I'm reading this as the opposite - vulnerable versions are all 3 versions up to 3.9.0? 3.9 containers look like they're on 3.9.12

**Pull request merged**